pyproject.toml

```diff
@@ -23,3 +23,9 @@ vcs = "git"
 [build-system]
 requires = ["poetry-core>=1.0.0", "poetry-dynamic-versioning"]
 build-backend = "poetry_dynamic_versioning.backend"
+
+[tool.poetry.plugins."psychopy.experiment.components"]
+ResponsepixxComponent = "psychopy_pixx:ResponsepixxComponent"
```

Comment:
Instead of from the top-level module, you can add an entry point to the components module. E.g. something like:
```
ResponsepixxComponent = "psychopy_pixx.components.responsepixx:ResponsepixxComponent"
ViewpixxSetupRoutine = "psychopy_pixx.routines.viewpixx:ViewpixxSetupRoutine"
```